census-api/src/main/scala/hmda/census/records/CensusRecords.scala

```diff
@@ -0,0 +1,55 @@
+package hmda.census.records
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.census.Census
+import hmda.model.ResourceUtils._
+
+trait CensusRecords {
```

Comment:
It's a matter of preference, but this should probably be an `object`, not a `trait`. That way it becomes a singleton, and you probably don't want to extend / override the functions defined here anyway